src/WallsComputation.cpp

```diff
@@ -129,15 +129,15 @@ void WallsComputation::generateInsets(SliceLayerPart* part)
 
     constexpr float transitioning_angle = 0.5;
 
-    Polygons tubeshape = part->outline.tubeShape(line_width_0 + line_width_x * (inset_count - 1), 0).offset(-10).offset(10);
-    tubeshape.simplify(50, 50);
-    tubeshape.removeColinearEdges(0.03);
-    tubeshape.fixSelfIntersections();
-    tubeshape.removeSmallAreas(INT2MM(line_width_0/2) * INT2MM(line_width_0/2), false); // TODO: complete guess as to when arachne starts breaking, but it doesn't function well when an area is really small apearantly?
-    if (tubeshape.area() > 0)
+    Polygons prepared_outline = part->outline.offset(-10).offset(10);
+    prepared_outline.simplify(50, 50);
+    prepared_outline.removeColinearEdges(0.03);
+    prepared_outline.fixSelfIntersections();
+    prepared_outline.removeSmallAreas(INT2MM(line_width_0/2) * INT2MM(line_width_0/2), false); // TODO: complete guess as to when arachne starts breaking, but it doesn't function well when an area is really small apearantly?
```

Comment:
I see `line_width_0/2' being calculated multiple times, why not calculate it once and store it in a const var?

Reply:
Because I copied it over from somewhere else I think. Shame, I'm usually the first to remove such double calcs!